File: aifupan_project/fupan-server/replay-words/src/main/java/com/jiuyu/replay/words/entity/AnchorUrlEntity.java
package com.jiuyu.replay.words.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

import java.io.Serializable;
import java.util.Date;

/**
 * 主播url
 *
 * @author jxy
 * @email 1776764427@qq.com
 * @date 2024-08-10 16:47:37
 */
@Data
@TableName("tb_anchor_url")
public class AnchorUrlEntity implements Serializable {
	private static final long serialVersionUID = 1L;

	/**
	 * ID
	 */
	@TableId(value = "id", type = IdType.INPUT)
	private Long id;
	/**
	 * 主播唯一标识
	 */
	private String secUid;
	/**
	 * 主页url
	 */
	private String homeUrl;
	/**
	 * 直播间url
	 */
	private String liveUrl;
	/**
	 * 主播名称
	 */
	private String anchorName;
	/**
	 * 主播头像
	 */
	private String anchorAvatar;
	/**
	 * 平台类型 0：抖音 1：快手 2：视频号
	 */
	private Integer platform;
	/**
	 * DouYinHomeLive(个人主页地址),DouYinLive（直播地址） 抖音直播 TiktokLive 国外版本抖音  KuaiShouLive 快手   其他待定
	 */
	private String platformResource;
	/**
	 * 主播userId
	 */
	private String anchorUserId;
	/**
	 * webSocketId
	 */
	private String webSocketId;
	/**
	 * 主播抖音号
	 */
	private String anchorNumber;
	/**
	 * 第三方数据平台是否已经收录 0:否 1:是
	 */
	private Integer chanmamaInclude;
	/**
	 * 系统行业id
	 */
	private Long systemTradeId;
	/**
	 * AI纠正后的行业id
	 */
	private Long aiCorrectTradeId;
	/**
	 * 添加AI纠正行业的时间
	 */
	private Date addAiTradeDate;
	/**
	 * 最后修改系统行业的时间
	 */
	private Date updateSystemTradeDate;
	/**
	 * 创建时间
	 */
	private Date createDate;
	/**
	 * 最后修改时间
	 */
	private Date updateDate;

}

File: aifupan_project/ReviewAnalysis/Asr/AnchorUrlEntity.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ReviewAnalysis.Asr
{
    public class AnchorUrlEntity
    {
        /// <summary>
        /// ID
        /// </summary>
        public long Id { get; set; }

        /// <summary>
        /// 主播唯一标识
        /// </summary>
        public string SecUid { get; set; }

        /// <summary>
        /// 主页URL
        /// </summary>
        public string HomeUrl { get; set; }

        /// <summary>
        /// 直播间URL
        /// </summary>
        public string LiveUrl { get; set; }

        /// <summary>
        /// 主播名称
        /// </summary>
        public string AnchorName { get; set; }

        /// <summary>
        /// 主播头像
        /// </summary>
        public string AnchorAvatar { get; set; }

        /// <summary>
        /// 平台类型 0：抖音 1：快手 2：视频号
        /// </summary>
        public int Platform { get; set; }

        /// <summary>
        /// DouYinHomeLive(个人主页地址), DouYinLive（直播地址） 抖音直播 TiktokLive 国外版本抖音  KuaiShouLive 快手   其他待定
        /// </summary>
        public string PlatformResource { get; set; }

        /// <summary>
        /// 行业id
        /// </summary>
        public string TradeId { get; set; }

        /// <summary>
        /// 主播userId
        /// </summary>
        public string AnchorUserId { get; set; }

        /// <summary>
        /// webSocketId
        /// </summary>
        public string WebSocketId { get; set; }

        /// <summary>
        /// 是否从录制列表移除了 0：否 1：是
        /// </summary>
        public int IsRemoveRecord { get; set; }

        /// <summary>
        /// 是否保存弹幕 0保存，1不保存
        /// </summary>
        public int IsBarrageMonitoring { get; set; }

        /// <summary>
        /// 检测在线直播时，是否录制视频 is_auto_record  0否，1是
        /// </summary>
        public int IsAutoRecord { get; set; }

        /// <summary>
        /// 是否自动上传到云空间 0：否 1：是
        /// </summary>
        public int IsAutoUploadCloud { get; set; }

        /// <summary>
        /// 是否置顶 0：否 1：是
        /// </summary>
        public int IsTop { get; set; }

        /// <summary>
        /// 加入置顶的时间
        /// </summary>
        public string AddTopTime { get; set; }
        /// <summary>
        /// 最后开始录制时间
        /// </summary>
        public string LastRecordTime { get; set; }
    }
}
